data/helpers.d/filesystem

```diff
@@ -80,3 +83,23 @@ properly with chmod/chown." >&2
     chmod 755 $TMP_DIR
     echo $TMP_DIR
 }
+
+# Restore a file or a directory to the stored path
+#
+# Use the registered path in backup_list by ynh_backup to restore the file at the good place.
+#
+# usage: ynh_restore_file file_to_restore
+# | arg: file_to_restore - File or directory to restore
+ynh_restore_file () {
```

Comment:
Once helper to restore all files is a bad idea I think.
First, you lose all your flexibility because you lost all possibility to restore like you want with some actions between each restore.

And second, you can't see what is restored and when. For debugging, I think it's really a bad idea.

Too much abstraction kill abstraction !
We lose readability, we lose flexibility, and we lose what it's really happened in the script.

Don't do too much abstraction !